### DISCUSSION TITLE
Status of the project

### DISCUSSION THREAD

**Discussion by bison** [aws/aws-cloud-map-mcs-controller-for-k8s#295] [General]
Hi, I just wanted to ask about the status of this project. The last release was a while ago, and the Slack linked in the README doesn't seem to be accessible to outside users. Can we expect further development on this, or has something maybe superseded it? I know there is also multi-cluster support via the Gateway Controller and VPC Lattice, but this seems like a great solution for single-VPC multi-cluster setups.

**Comment by runakash** [comment]
Hello @bison - We are in maintenance mode. Will work on critical bug fixes but not on releasing any new features. We have plans for releasing milestone 4, but nothing concrete yet.
